docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
It probably makes sense to also commit the `Makefile` generated by sphinx to the repo, since we're more likely to build on Mac/Linux than Windows.

Reply:
Never mind, it's already in there.